Dockerfile

```diff
@@ -19,13 +19,18 @@ RUN --mount=source=.,target=/mnt,type=bind \
 
 RUN curl -L --insecure https://trex-tgn.cisco.com/trex/release/v3.05.tar.gz | tar -xz -C /opt/trex
 
-RUN curl https://raw.githubusercontent.com/gkbrk/slowloris/refs/heads/master/slowloris.py -o /usr/local/bin/slowloris \
- && chmod +x /usr/local/bin/slowloris
-
 RUN --mount=source=.,target=/mnt,type=bind \
     cp -r /mnt/src /var/www/html/secflood \
  && install --mode 0755 --owner root /mnt/scripts/trex /usr/local/bin/
 
+RUN curl https://raw.githubusercontent.com/gkbrk/slowloris/refs/heads/master/slowloris.py -o /usr/local/bin/slowloris \
+ && chmod +x /usr/local/bin/slowloris
+
+RUN apt-get update \
+ && apt install --no-install-recommends -y libnet-ssleay-perl \
```

Comment:
Suggest to install this dependency along with the other debian packages (line 6) or to remove the apt cache (as per line 12)

Reply:
Great!! Fixed